jupyter_server_proxy/handlers.py

```diff
@@ -316,7 +316,9 @@ def get_env(self):
                 # Set up extra environment variables for process
                 server_env.update(self.get_env())
 
-                proc = SupervisedProcess(self.name, *cmd, env=server_env, ready_func=self._http_ready_func, log=self.log)
+                timeout = self.get_timeout()
```

Comment:
Can you provide a default implementation of this too, since people might wanna use this handler by itself?